api/v1alpha1/webhook_suite_test.go

```diff
@@ -73,40 +73,91 @@ func TestAPIs(t *testing.T) {
 	RunSpecs(t, "Webhook Suite", suiteConfig, reporterConfig)
 }
 
-var _ = Context("Releases signatures are validated against the correct public key", func() {
```

Comment:
Can you move your new tests to the end? This really confused git's diff.

Reply:
Moved my tests to the end.